src/main/java/seedu/duke/Duke.java

```diff
@@ -1,21 +1,30 @@
 package seedu.duke;
 
+import seedu.duke.exceptions.InvalidCommandException;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.userinterface.CliUserInterface;
+
 import java.util.Scanner;
 
 public class Duke {
+    public static boolean notQuit = true;
+    private static final TaskList list = new TaskList();
+    private static int numberOfTasks = 0;
+
     /**
      * Main entry-point for the java.duke.Duke application.
      */
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
-
+    public static void main(String[] args) throws Exception {
+        CliUserInterface ui = new CliUserInterface(list, numberOfTasks);
         Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+        ui.startUI();
+        while (notQuit) {
```

Comment:
Do you think this is better here, or in the CliUserInterface class? I did do it this way in my iP, but I'm not sure this is necessarily the better way to do it.

Reply:
I did something similar in my own ip as well. xD See if the others have anything to say about it then?